### DISCUSSION TITLE
Proposal about 4 common (default) relays on Nostros

### DISCUSSION THREAD

**Discussion by serrq** [KoalaSat/nostros#321] [Ideas]
Since relays don't speak each other, they act like local communities. This is bad, by design.

As Nostros community I propose 4 relays active by default in all Nostros apps, in order to find each other easily.

It is enough that one among these be active:

• wss://nostr.mado.io
• wss://nostr.massmux.com
• wss://nostr.mwmdev.com
• wss://nostr1.tunnelsats.com

All tested by me: high propagation of your events.

**Comment by KoalaSat** [comment]
Havigna default list of relays increases the centralisation, we had in the past a list of 3 but we are actively moving away from that model. Right now we assign randomly 5 relays from a list, and I the future the Resilience experiment should be able to assign the best choice.

In the mid time I think that the best choice is to include these 4 on the fallback list of relays, the bigger the better!

**Reply by serrq** [reply]
I agree with centralization, but then how do we solve the problem of finding ourselves?

If we don’t have at least one relay in common, we’re not together.

Another problem: why do relays, by design, not talk to each other? This is a big problem.

**Reply by KoalaSat** [reply]
There is a way to include your relays as part of the public key you share with others, this is pending to be implemented for Nostros. 

About relays, ATM there is nothing about how relays Interac to each other. I'm glad you mentioned it because I recently created a proposal for the nostr protocol that actually helps relays to have some kind of replication network 😄

**Comment by serrq** [comment]
1. Never heard about this way. What is the way name? Is it a NIP proposal?
2. This is fundamental because I can be connected to one only relay and being propagate by it to all nostr network (other relays).

**Reply by KoalaSat** [reply]
1.https://github.com/nostr-protocol/nips/blob/master/19.md#shareable-identifiers-with-extra-metadata it's the `nprofile`
2. Just posted it 1 hour ago 😄 https://github.com/nostr-protocol/nips/pull/245

**Reply by serrq** [reply]
I don’t know if I understand NIP correctly but I don’t want to join any group or list. Simply any node must relay the received event to the rest of the network, without conditions. End of story.

The way I mean is similar to multi-hop in star model networks.

**Comment by KoalaSat** [comment]
That's not what happens right now with paid relays right now , and the group is made by the relays not by the users

**Reply by KoalaSat** [reply]
Anyways, the idea is to create a network of replication, for this NIP clients are a secondary entity

**Comment by serrq** [comment]
All those nodes that instead of propagating events to others nodes play to rich and poor I hope that a future vote will send them all home.

Profit exists for everyone. If I am a creator, I have no problem redistributing part of my earnings with developers and relays operators. But this rich and poor shit just has to stop. Relays have to bounce the network events. 

If not Nostr, I immediately subscribe to the next social platform Solana, similar to the model relays seen here, but without the ruling elites.
